protocols/mdns/src/behaviour.rs

```diff
@@ -85,6 +85,9 @@ pub struct Mdns {
     /// Iface watcher.
     if_watch: IfWatcher,
 
+    /// Send a query.
+    query: bool,
```

Comment:
In case we do go with the extra boolean, this would need additional documentation.